src/ui/public/vis/vis.js

```diff
@@ -36,6 +46,7 @@ export default function VisFactory(Notifier, Private) {
 
     let schemas = type.schemas;
 
+    // NOTE: What is this doing?
```

Comment:
I think it was around 4.0 that this went in, and it was only to support people who had saved visualizations during the 4.0 betas. I would be pretty surprised if someone expects visualizations created in the 4.0 betas to open fine in 5.0...